lib/awsm.sh

```diff
@@ -1,6 +1,21 @@
+
 : ${FUZZY_FILTER="fzf"}
 SSH_BIN=$(which ssh)
 
+## Environment setup
+
+if [ -r ~/.awsm ]; then
+  echo "Reading user config...." >&2
+  . ~/.awsm
```

Comment:
`~/.awsm` is a folder currently, I am leaning towards `~/.awsm/config` as a global config.